app/views/users/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1> Single user will be displayed</h1>
```

Comment:
Actually, this `index` action as a convention will show all users and the `show` action will show a specific user. This means that this index.html.erb will show all users and not a single user.

Reply:
@bobb-Rob Good morning, I thought I was missing something Thanks for pointing it out , it would have been fine if you did not approve but thanks anyways